packages/datadog-instrumentations/src/net.js

```diff
@@ -17,8 +17,14 @@ const errorTCPCh = channel('apm:net:tcp:error')
 
 const connectionCh = channel(`apm:net:tcp:connection`)
 
-addHook({ name: 'net' }, net => {
-  require('dns')
+const names = ['net', 'node:net']
+
+addHook({ name: names }, (net, version, name) => {
+  if (name === 'net') {
+    require('dns')
+  } else {
+    require('node:dns')
+  }
```

Comment:
Why are we loading the dns module here again? Maybe we could use a comment to remember?

Reply:
I believe it's because otherwise net gets an uninstrumented instance & thus we miss the dns calls